### DISCUSSION TITLE
Any issue with Netbox public facing Demo site ?

### DISCUSSION THREAD

**Discussion by saifvs** [netbox-community/netbox#11188] [Q&A]
I have been trying to login with the default creds on the netbox demo page [here](https://demo.netbox.dev/login/), but it doesnt let me in. getting the below error.
I was able to login before.

![image](https://user-images.githubusercontent.com/66313945/207796542-bc35a935-3931-4bc5-bd8d-b69b48d0391a.png)

**Comment by jeremystretch** [comment]
I hadn't migrated the demo to the new v3.4 data prior to it being upgraded during its nightly rebuild. Should be working now. Thanks for the heads up.
